packages/waku/src/lib/plugins/vite-plugin-rsc-transform.ts

```diff
@@ -5,10 +5,25 @@ import { EXTENSIONS } from '../config.js';
 import { extname } from '../utils/path.js';
 import { parseOpts } from '../utils/swc.js';
 
-const collectExportNames = (mod: swc.Module) => {
+const collectExportNames = (
```

Comment:
Here are other issues:

1. non export sever action could cause runtime error
2. nested server action could cause runtime error

```ts
const actionsMap = {
  foo: () => {
     "use server"
// ...
  } 
}
```

Those are exist in vercel/ai/rsc source so we need to support them

Reply:
Okay, I missed such use cases.
This pattern, inline "use server", is only allowed in the server files, right?

Reply:
Hm, I made a test case #771, but it looks like it's transforming well.
Can you give me a failing case?

This is also tried:
```js
const actionsMap = createActions({
  foo: () => {
     "use server"
     // ...
  } 
})
```